example-fileupload/README.md

```diff
@@ -0,0 +1,507 @@
+
+
+# Appwrite + VueJS =❤️
+This example is to showcase [Appwrite's JS API](https://github.com/appwrite/sdk-for-js) with [VueJS](https://vuejs.org/) by creating a File Uploading Application where you can Upload files and See recently uploaded files.
+
+## Prerequisites
+
+-   A Recent Version of NodeJS
```

Comment:
Why do the words `Recent` and `Version` start with an uppercase letter?

Also maybe link to the nodejs website